softlearning/environments/adapters/dm_control_adapter.py

```diff
@@ -0,0 +1,155 @@
+"""Implements an adapter for DeepMind Control Suite environments."""
+
+from collections import OrderedDict
+
+import numpy as np
+from dm_control import suite
+from dm_control.rl.specs import ArraySpec, BoundedArraySpec
+from gym import spaces
+
+from .softlearning_env import SoftlearningEnv
+
+
+# TODO(hartikainen): Add information of the available environments.
```

Comment:
Let's ignore this until we add custom environments. I.e. remove the comment for now.